net/adblock/files/adblock.sh

```diff
@@ -613,6 +613,7 @@ f_list()
 						out_rc="${?}"
 					fi
 				fi
+				rm -f "${adb_tmpdir}/tmp.raw.${mode}"
```

Comment:
Can `tmp.add.whitelist` be removed here as well?

Reply:
Nope, it's part of the final list merge.